app/assets/javascripts/oxalis/model/accessors/flycam_accessor.js

```diff
@@ -1,4 +1,3 @@
-// @flow
```

Comment:
no more flow?

Reply:
Oops, this was meant to only change test-files. I'll fix it.